### PR TITLE
Bump dependencies, node engine, refactor with CommonJS and ES6

### PR DESCRIPTION
**Pull request by lndgalante**

### CODE REVIEW

**Comment by hdf1986** [comment]
:heart:

File: server/server.js [hunk 1]
```diff
@@ -1,162 +1,206 @@
...
+  const headers = Object.assign(
+    removeKeys(req.headers, FORBIDDEN_CLIENT_HEADERS),
+    {}
+  )
+
+  const k = request(url, { headers })
+
+  k
+    .on('response', r => {
+      r.headers['server'] = 'JAY PENZOIL DRINKERS HOGO V1'
```
Comment (hdf1986) [inline server/server.js]: You need to change this to V2 and add some LEO

File: server/server.js [hunk 2]
```diff
@@ -1,162 +1,206 @@
...
-
-app.put(/(http|https)(:)\/\/(.*)/, function (req, res) {
-  var url = obtainUrl(req);
-  console.log("Request url PUT: " + url)
+    headers: headers,
+  })
+
+  k
+    .on('response', r => {
+      r.headers['server'] = 'JAY PENZOIL DRINKERS HOGO V1'
```
Comment (hdf1986) [inline server/server.js]: idem

File: server/server.js [hunk 3]
```diff
@@ -1,162 +1,206 @@
...
-
-app.delete(/(http|https)(:)\/\/(.*)/, function (req, res) {
-  var url = obtainUrl(req);
-  console.log("Request url DELETE: " + url)
+    headers: headers,
+  })
+
+  k
+    .on('response', r => {
+      r.headers['server'] = 'JAY PENZOIL DRINKERS HOGO V1'
```
Comment (hdf1986) [inline server/server.js]: idem

File: server/server.js [hunk 4]
```diff
@@ -1,162 +1,206 @@
...
-app.use(express.static('public'));
-
-app.listen(app.get('port'), function () {
-  console.log('CORS Running on port 5000!');
-});
+  })
+
+  k
+    .on('response', r => {
+      r.headers['server'] = 'JAY PENZOIL DRINKERS HOGO V1'
```
Comment (hdf1986) [inline server/server.js]: idem

File: server/server.js (hunk 1 above)
Comment (lndgalante) [reply server/server.js]: love this CR 🎉

**Pull request merged**